### ISSUE TITLE
Intercom setup - change to rewardseverywhere.co ?

### ISSUE THREAD

**Issue by cgladue** [robgpeak/CompLinks-xclub#267]
Any changes to intercom to support the new domain ?

**Comment by robgpeak** [comment]
Not really, when we send mail out if intercom, it uses their domain.  We might need to redirect the incoming mail from info@rewardseverywhere.co and to point to Intercom's servers. Or just send a copy there.  I will have to look at it tomorrow.

**Comment by robgpeak** [comment]
As I mention in another issue, I have the email sorted and it is going to Intercom.

On this page https://app.intercom.io/a/apps/d8m2yyar/settings/web it describes how to put the code into a new site, i.e. rewardseverywhere.co

I don't recall if there was a section in Intercom that requires you to "add a domain" that your chat widget will be on.

As for our internal records in Intercom, xclub.complinks.co and xclub.rewardseverywhere.com would still have the same "Company Number and Name" Xclub
